servers/cromwell/README.md

```diff
@@ -40,26 +40,29 @@ Thin shim around [`cromwell`](https://github.com/broadinstitute/cromwell).
       "display": "Workflow ID"
     },
     {
-     "field": "labels.flag",
-     "display": "Flag",
-     "editable": true,
-     "fieldType": "list",
-     "fieldValues": [
-       "archive",
-       "follow-up"
-     ]
-   },
-   {
-     "field": "labels.label",
-     "display": "Label",
-     "fieldType": "text",
-     "editable": true
-   },
-   {
-     "field": "labels.comment",
-     "display": "Comment",
-     "fieldType": "text",
-     "editable": true
+      "field": "labels.flag",
+      "display": "Flag",
+      "editable": true,
+      "bulkEditable": true,
+      "fieldType": "list",
+      "fieldValues": [
```

Comment:
Please change this to `validFieldValues`

Reply:
done – thanks for catching that!